demisto_sdk/commands/init/initiator.py

```diff
@@ -31,12 +31,67 @@ class Initiator:
            name (str): The name for the new pack/integration/script directory.
            id (str): The id for the created script/integration.
            integration (bool): Indicates whether to create an integration.
+           template (str): If an integration is initialized, specifies the integration template.
            script (bool): Indicates whether to create a script.
            full_output_path (str): The full path to the newly created pack/integration/script
     """
 
-    def __init__(self, output: str, name: str = '', id: str = '', integration: bool = False, script: bool = False,
-                 pack: bool = False, demisto_mock: bool = False, common_server: bool = False):
+    TEST_DATA_DIR = 'test_data'
+
+    ''' INTEGRATION TEMPLATES CONSTANTS '''
+    DEFAULT_INTEGRATION_TEMPLATE = 'BaseIntegration'
```

Comment:
what is the difference between BaseIntegration and HelloWorld ? We better think more user friendly options

Reply:
HeloWorld is the template that was so far, BaseIntegration is the new one made by @fvigo. I guess this was discussed before, but anyway if we decide to change their names this should happen in content repo first and we can open a separate PR for it here later

Reply:
@anara123 - This is a more slimmed-down initial integration since HelloWorld is robust and easy to lose yourself in.
This one creates an easier and cleaner starting point and consulting with @fvigo it seemed like something that would benefit contributors more.